L1Trigger/TrackFindingTracklet/interface/PurgeDuplicate.h

```diff
@@ -40,19 +40,15 @@ namespace trklet {
     bool isSeedingStub(int, int, int);
     std::string l1tinfo(const L1TStub*, std::string);
     std::pair<int, int> findLayerDisk(const Stub*);
-    std::vector<double> getInventedCoords(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for prompt tracking
-    std::vector<double> getInventedCoordsExtended(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for extended tracking
-    std::vector<const Stub*> getInventedSeedingStub(
-        unsigned int,
-        Tracklet*,
-        std::vector<const Stub*>);  // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
-
+    // calculate stub coordinates based on tracklet trajectory for prompt tracking
```

Comment:
Any function that doesn't change class data members must be declared "const". e.g.
bool isSeedingStub(int, int, int) const;
Most functions in this class (including those not written by you) ignore this rule. I'd appreciate if you could fix this.

Reply:
This has been done.